hack/setup_env.sh

```diff
@@ -48,7 +48,7 @@ function test_tools() {
   go install github.com/golang/mock/mockgen@v1.6.0
   go install github.com/vektra/mockery/v2@v2.12.3
   go install gotest.tools/gotestsum@v1.6.3
-  go install github.com/axw/gocov/gocov@latest
+  go install github.com/axw/gocov/gocov@v1.1.0
   go install github.com/AlekSi/gocov-xml@v0.0.0-20190121064608-3a14fb1c4737
```

Comment:
Seems like we need to pin also `gocov-xml` version.
Try it on: https://github.com/openshift/assisted-service/pull/6902

Reply:
Thanks for catching this and adding it in!! 🙌